views/choral.pug

```diff
@@ -35,10 +35,10 @@ block content
             div.child-card-body
               - var device_data = JSON.parse(child.device_data);
               - for( var entry in device_data )
-                - var data_value = device_data[entry]
+                - var data_value = device_data[entry].toFixed(5)
```

Comment:
Probably shouldn't do this on the server as that would only affect the first value.